_projects/enhance-and-improve-coala-community-and-projects-website.md

```diff
@@ -0,0 +1,160 @@
+
+    c. Add [contributor's Geolocation](https://github.com/coala/community/issues/18)
+
+    d. Add [Organisation Teams](https://github.com/coala/community/issues/12)
+
+    e. Add [Mentors](https://github.com/coala/community/issues/11)
+
+    f. Add **Newcomer Invitation** [form](https://github.com/coala/community/issues/89#issuecomment-468955279)
+
+    g. Add **Promotion to Developer** [form](https://github.com/coala/community/issues/89)
+
+    h. Manage live data of community members
+
+    i. Addition of more data fields in **Users** db model
+
+    j. Add **Login and Sign-up** using GitHub
+3. Make Improvements (or fix bugs) in coala [community website](https://community.coala.io/)
+
+    a. Build failure for more than one remote [#235](https://github.com/coala/community/issues/235)
+
+    b. Netlify build failure for fork [#232](https://github.com/coala/community/issues/232)
+
+#### CODING PHASE 2
+
+1. Implementation of listed enhancements in coala [community website](http://community.coala.io/)
+
+    a. Create **Profile Page**
+
+    b. Leaderboard on **[Homepage](https://coala-community.netlify.com/)**
+
+    c. Enhancement in **[Gamification Leaderboard](https://coala-community.netlify.com/gamification/)**
+
+2. Implementation of listed enhancements in [projects website](https://projects.coala.io)
+
+    a. Add **[Filters](https://github.com/coala/projects/issues/559)** for searching projects
+
+3. Make Improvements in coala [community website](https://community.coala.io/)
+
+    a. **[Gamification Leaderboard](https://coala-community.netlify.com/gamification/)**
+
+    b. **[UI](https://coala-community.netlify.com/)**
+      * [Re-designing of the website](https://github.com/coala/community/issues/44) and making it responsive.
+      * [Inactive Issue](https://coala-community.netlify.com/static/inactive-issues.json) and [unassigned issues activity](https://coala-community.netlify.com/static/unassigned-issues.json) web pages shows
+        just JSON response, it should be displayed in proper format with some
+        extra important information.
+      * [Meta-Review data](https://coala-community.netlify.com/meta-review/) web pages should be redesigned to make it look good.
+
+#### CODING PHASE 3
+
+1. Implementation of listed enhancements in [projects website](https://projects.coala.io)
+
+    a. [Improve](https://github.com/coala/projects/issues/394) the projects website for use by other GSoC orgs
+
+    b. Create [tab for all the google forms](https://github.com/coala/projects/issues/284)
+
+    c. Add [posts / calendar app](https://github.com/coala/projects/issues/560) to projects website
+
+    d. Create a tab for **GSoC Students**
+
+2. Implementation of listed enhancements in coala [community website](http://community.coala.io/)
+
+    a. Create a feedback form
+
+    b. Create a databse model for feedback entries
+
+    c. Automating the process of assigning issues
```

Comment:
How you plan to do this?

Reply:
I was planning to achieve this by creating a function that will perform validation checks to avoid spam. And automating this process of assigning requires the use of OAuth token(with issue edit rights) created by the organization. 
The token can be added by only members of `coala-maintainers` team from there profile (Which will be acting as a dashboard for them) via a form type UI.

Reply:
An option will be provided to user `want to work on issue?` choosing it will make an API call at coala-webservices which will perform validation checks. If all checks got validated the user will be assigned on that issue.

Reply:
We already have functionality of automating assigning of issues in corobo.

Reply:
Yes, we're having that functionality but currently `corobo` is down for maintenance and will be up mp after GSoC'19. And moreover, it doesn't prevent spam i.e. anyone can get assigned on an issue without performing some validation checks i.e. whether the user has solved the previous issue or not, has newcomer completed newcomers guide before getting assigned on the high-level issue and many more. 
I want to add this because it will include more security which can prevent spam.